packages/mux-player/src/index.ts

```diff
@@ -484,6 +497,22 @@ class MuxPlayerElement extends VideoApiElement {
   set metadata(val) {
     if (this.video) this.video.metadata = val;
   }
+
+  get tokens() {
+    const playback = this.getAttribute(PlayerAttributes.PLAYBACK_TOKEN);
+    const thumbnail = this.getAttribute(PlayerAttributes.THUMBNAIL_TOKEN);
+    const storyboard = this.getAttribute(PlayerAttributes.STORYBOARD_TOKEN);
+    return {
+      ...this.#tokens,
+      ...(playback != null ? { playback } : {}),
```

Comment:
seems reasonable to me. We could optimize this for the null cases, but I don't think that's a huge priority.